### ISSUE TITLE
what is your training parameters?

### ISSUE THREAD

**Issue by Johnson-yue** [taki0112/StarGAN_v2-Tensorflow#5]
Hi , I was reimplementing this work but using Pytorch ,and result is very bad.
thank you for your sharing ,and I have some questions :
1. what is your training parameters , like "gan_type": 
  you default gan_type is gan but this type not using gradient panalty ,and author said using R1 GP, so I guess you used gan_type=dragan ? right?

2. How do you get target_domain label y~ and source_domain label y ?
   in my opinion,  the dataloader about this model  is more likely classification training.
  data format is [X_img, domain_label ]
  I think domain_label is original domain of x , so it is  source_domain label y. But what is target_domain label y~ and how to get it?? just random ??

**Comment by taki0112** [comment]
Hello !
I will re-upload the tensorflow2 version of stargan v2 to [clova_ai](https://github.com/clovaai/stargan-v2-tensorflow)

If you have any questions, use the above link ! 
Thank you.

**Comment by Johnson-yue** [comment]
Thank you
